config/config.go

```diff
@@ -431,7 +423,7 @@ type (
 // New creates a config instance. It first loads the default configs. If the config path is not empty, it will read from
 // the file and override the default configs. By default, it will apply all validation functions. To bypass validation,
 // use DoNotValidate instead.
-func New(validates ...Validate) (Config, error) {
+func New(_overwritePath string, _secretPath string, validates ...Validate) (Config, error) {
```

Comment:
merge _overwritePath and _secretPath into `configPaths []string`. in line 430, loop through these paths:
```
    for _, path in range configPaths {
        if path != "" {
            opts = append(opts, uconfig.File(path))
        }
    }
```